### PR TITLE
added support for disabled options in em-select

### PR DESCRIPTION
**Pull request by bluemari**

### CODE REVIEW

File: addon/components/em-select.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ Syntax:
     optionValuePath=keyForValue
     optionLabelPath=keyForLabel
     prompt="Optional default prompt"}}
-
+    optionDisabledPath=keyForDisabled
```
Comment (pedrokiefer) [inline addon/components/em-select.js]: Your new argument is outside the end of the tag `}}`.

You also missed to define it as null on the body of FormGroupComponent.

File: addon/templates/components/html-select.hbs [hunk 2]
```diff
@@ -5,7 +5,7 @@
   </option>
 {{/if}}
 {{#each (get mainComponent "content") key="@index" as |item|}}
-  <option value="{{get item mainComponent.optionValuePath}}" selected={{eq (get item mainComponent.optionValuePath) selectedValue}}>
+  <option value="{{get item mainComponent.optionValuePath}}" selected={{eq (get item mainComponent.optionValuePath) selectedValue}} disabled={{get item mainComponent.optionDisabledPath}}>
```
Comment (pedrokiefer) [inline addon/templates/components/html-select.hbs]: Could you also write a couple of test cases for this?

**Comment by GCorbel** [comment]
@bluemari any update?

**Comment by spruce** [comment]
Closed in favor of #173

**Pull request closed without merging**